src/modules/category/selectors.ts

```diff
@@ -1,24 +1,74 @@
 import { createSelector } from 'reselect'
 import { RootState } from 'modules/common/types'
 import { CategoryState } from 'modules/category/reducer'
-import { AssetState } from 'modules/asset/reducer'
 import { FullCategory } from 'modules/category/types'
+import { getData as getAssetPacks } from 'modules/assetPack/selectors'
 import { getData as getAssets } from 'modules/asset/selectors'
+import { AssetState } from 'modules/asset/reducer'
+import { AssetPackState } from 'modules/assetPack/reducer'
+import { Asset } from 'modules/asset/types'
 
 export const getState: (state: RootState) => CategoryState = state => state.category
 
-export const getData: (state: RootState) => CategoryState['data'] = state => getState(state).data
+export const getSelectedAssetPackId = (state: RootState) => getState(state).selectedAssetPackId
```

Comment:
I don't get why is this here, wouldn't this be a responsibility of assetPack?

Reply:
If i recall correctly the idea was to leave the possibility to choose a single asset pack at a time, while `null` would merge all assets packs together by category.

This is one of those things that might get rolled back in the future depending on the direction we take. What do you think?

Reply:
Maybe `category` is not the right name anymore, but the responsibility of this module is to generate the `FullCategory[]` array that we use to render the item drawer. 

We could rename it to `ui` or `sidebar` or something else, but i think the selected asset pack id and search string belong to a separate module rather that `assetPack`.

Reply:
> We could rename it to ui or sidebar or something else, but i think the selected asset pack id and search string belong to a separate module rather that assetPack.

Agreed

> Maybe category is not the right name anymore, but the responsibility of this module is to generate the FullCategory[] array that we use to render the item drawer.

Think so, I think the only problem here (if there's one) is the name of the domain. I like both UI and sidebar

Reply:
refactored into `ui.sidebar`